eitprocessing/parameters/eeli.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import dataclass, field
+from typing import Literal, get_args
+
+import numpy as np
+
+from eitprocessing.datahandling.continuousdata import ContinuousData
+from eitprocessing.features.breath_detection import BreathDetection
+from eitprocessing.parameters import ParameterCalculation
+
+
+@dataclass
+class EELI(ParameterCalculation):
+    """Compute the end-expiratory lung impedance (EELI) per breath."""
+
+    method: Literal["breath_detection"] = "breath_detection"
+    breath_detection_kwargs: dict = field(default_factory=dict)
+
+    def __post_init__(self):
+        _methods = get_args(EELI.__dataclass_fields__["method"].type)
+        if self.method not in _methods:
+            msg = f"Method {self.method} is not valid. Use any of {', '.join(_methods)}"
+            raise ValueError(msg)
+
+    def compute_parameter(self, continuous_data: ContinuousData, sample_frequency: float) -> np.ndarray:
+        """Compute the EELI per breath.
+
+        Example:
+        >>> global_impedance = sequence.continuous_data["global_impedance_(raw)"]
+        >>> sample_frequency = sequence.eit_data["raw"].framerate
+        >>> eeli_values = EELI().compute_parameter(global_impedance, sample_frequency=sample_frequency)
+
+        Args:
+            continuous_data: a ContinuousData object containing the data.
```

Comment:
Should it be explicit that the ContinuousData object should be an impedance signal? Otherwise the EELI might not be the proper name for the entire parameter?

Reply:
Yes!

I think we should discuss and properly implement the `category` system for this. We added a `category` attribute, so that we can indicate what type of data is in a data container. We should make sure this function only works if the category is e.g. `"impedance"`.

Let's discuss categories and implement them before releasing this feature?

Reply:
Improved docstring in 66c2d6ad7dabd77aa0b3a1ec4838972f7110ea46